crates/header-translator/src/stmt.rs

```diff
@@ -579,7 +579,7 @@ impl Stmt {
                                 cls: id.clone(),
                                 generics: generics.clone(),
                                 category: ItemIdentifier::with_name(None, entity, context),
-                                availability: Availability::parse(entity, context),
+                                availability: availability.clone(),
```

Comment:
I don't know that this is the correct way to deal with this. I noticed the generated cfg attributes were gone for superclass functions. I'm guessing that the `Availability::parse(entity, context)` here checked the availability of the super not the subclass.